### PR TITLE
chore: add additional testcases for fswap

### PR DESCRIPTION
**Pull request by jaeseung-bae**
## Description

* add more tests for fswap

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/fswap/keeper/grpc_query_test.go [hunk 1]
```diff
@@ -42,17 +43,17 @@ func (s *FSwapQueryTestSuite) SetupTest() {
...
 	s.keeper = s.app.FswapKeeper
 
-	pebSwapRateForCony, err := sdk.NewDecFromStr("148079656000000")
+	kaiSwapRateForCony, err := sdk.NewDecFromStr("148079656000000")
 	s.Require().NoError(err)
 	swapCap := sdk.NewInt(1000)
 	s.Require().NoError(err)
 	s.fromDenom = "cony"
-	s.toDenom = "kei"
+	s.toDenom = "kai"
```
Comment (zemyblue) [inline x/fswap/keeper/grpc_query_test.go:51]: `kei` is right.

File: x/fswap/keeper/genesis_test.go [hunk 2]
```diff
@@ -45,3 +51,78 @@ func (s *KeeperTestSuite) TestInitAndExportGenesis() {
...
+						Amount: sdk.ZeroInt(),
+					},
+					ToCoinAmount: sdk.Coin{
+						Denom:  "bbb",
+						Amount: sdk.ZeroInt(),
+					},
+				},
+			},
+		}
+	}
```
Comment (170210) [inline x/fswap/keeper/genesis_test.go:83]: Could this function as a private function of `genesis_test` and also used in `TestInitAndExportGenesis()`

File: x/fswap/keeper/grpc_query_test.go [hunk 3]
```diff
@@ -88,48 +89,229 @@ func (s *FSwapQueryTestSuite) SetupTest() {
...
+		},
+		{
+			name: "invalid request",
+			request: &types.QuerySwapsRequest{
+				Pagination: &query.PageRequest{
+					Key:        []byte("invalid-key"),
+					Offset:     1,
+					Limit:      0,
+					CountTotal: false,
+					Reverse:    false,
```
Comment (170210) [inline x/fswap/keeper/grpc_query_test.go:304]: I think these two lines could be deleted. (Other test functions which test pagination haven't these options)

File: x/fswap/types/msgs_test.go [hunk 4]
```diff
@@ -89,49 +92,328 @@ func TestAminoJSON(t *testing.T) {
 func TestQuerySwapRequestValidate(t *testing.T) {
```
Comment (170210) [inline x/fswap/types/msgs_test.go:92]: I think these part is duplicated in `msg_server_test`.
Could we standardize the validation of messages? Except for the swap, the other three messages do not have a validation function. I suggest that QuerySwapRequest could also be validated similarly. And currently it only checks if the value is null without using sdk.ValidateDenom, which I think is insufficient.

Comment (jaeseung-bae) [reply x/fswap/types/msgs_test.go:92]: After I checked duplication, `TestQuerySwapRequestValidate` is duplicated and redundant.
Thank you, I'll remove it

> And currently it only checks if the value is null without using sdk.ValidateDenom, which I think is insufficient.
I agree with it. 
I want to add test cases only and don't want to change production code. We may update insufficient validation logic in other PR.

**Review by zemyblue — APPROVED** [review]

**Review by 170210 — APPROVED** [review]

**Pull request merged**